### ISSUE TITLE
gtcheck and haploid

### ISSUE THREAD

**Issue by sachalau** [samtools/bcftools#872]
Hello,

I was trying to use gtcheck for haploid data. I skimmed through the code of the  function and saw various mentions  about diploidy. Can you confirm that gtcheck does not work for anything other than diploid ? Because I am seeing only weird results. I did not see this specified on the doc or looking in the issues of this github. I am using the option --GTs-only 1.

Thanks a lot

**Comment by pd3** [comment]
That's correct, the program works with diploid genotypes only at the moment.
